addons/binding/pom.xml

```diff
@@ -3,16 +3,16 @@
 
   <modelVersion>4.0.0</modelVersion>
 
-  <parent>
```

Comment:
It would be great if you could revert your changes, in this file. I don't know what we want but it seemed that this file contained spaces instead of tabs :-)

Reply:
Might got changed automatically when merging. Will update it with spaces again.